Watchman.Discord/Areas/Protection/RequestParsers/WarnRequestParser.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Devscord.DiscordFramework.Commons.Exceptions;
+using Devscord.DiscordFramework.Framework.Commands.Parsing.Models;
+using Devscord.DiscordFramework.Middlewares.Contexts;
+using Devscord.DiscordFramework.Services;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Watchman.DomainModel.Messages;
+using Watchman.DomainModel.Users;
+
+namespace Watchman.Discord.Areas.Protection.Services
+{
+    public class WarnRequestParser : RequestParser
```

Comment:
co masz z dziedziczenia po RequestParserze?

Reply:
aa widze, to twoja klasa

Reply:
GetUser